### DISCUSSION TITLE
Adjustable number of QSOs visible in logbook view

### DISCUSSION THREAD

**Discussion by ofadam** [magicbug/Cloudlog#553] [Ideas / Feature Requests]
On the logbook view, it would be nice to be able to adjust how many QSOs is shows per page - or even have all on one page. That would let you see all plotted on the map (is this available somewhere else?)

**Comment by myzinsky** [comment]
Totally agree :)

**Comment by magicbug** [comment]
Not likely anytime soon but can go on the features list
